WeakAurasOptions/RegionOptions/Icon.lua

```diff
@@ -191,81 +195,107 @@ local function createOptions(id, data)
       width = WeakAuras.normalWidth,
       name = L["Enable Swipe"],
       order = 11.1,
-      desc = L["Enable the \"Swipe\" radial overlay"],
-      disabled = function() return not OptionsPrivate.Private.CanHaveDuration(data); end,
-      get = function() return OptionsPrivate.Private.CanHaveDuration(data) and data.cooldown; end
+      desc = L['Enable the "Swipe" radial overlay'],
```

Comment:
Such changes (swap double quotes to single ones) will break the automatic update of translation (and, by the way, will delete phrases already pushed to WowAce). Because Babelfish script can't parse apostrophe `'`. This has already been discussed. See PR [3476](https://github.com/WeakAuras/WeakAuras2/pull/3476) and [this](https://discord.com/channels/172440238717665280/273144328447197186/940010228437647390) thread

Reply:
We can force it to always be double quotes

Reply:
We can force it to always be double quotes